src/main/kotlin/com/cognifide/gradle/aem/instance/InstanceSync.kt

```diff
@@ -484,6 +486,30 @@ class InstanceSync(aem: AemExtension, instance: Instance) : InstanceHttpClient(a
         return scripts.asSequence().map { evalGroovyScript(it, data, verbose) }
     }
 
+    override fun getNode(path: String): Node? {
+        return Node.load(this, path)
+    }
+
+    override fun createNode(path: String, props: Map<String, Any>): Node {
```

Comment:
please test and handle properly null values in maps. probably putting null value should remove property from node. please ensure that API will handle that case

Reply:
currently such case is not supported ib `HttpClient`
`fun post(url: String, params: Map<String, Any> = mapOf()) = postUrlencoded(url, params)`

Should I change it or create new issue for backlog